docu/SICKAppSpaceCodingStarterKit_Documentation.md

```diff
@@ -72,9 +88,9 @@ Now you have different options:
 
```

Comment:
Optional / Question: is there somwhere a list or examples of these use cases where UI interaction is enough? Could you link them here or name some in brackets? (e.g. image...task by combining the following modules: ..., ..., ....)